### PR TITLE
Add test for EACH_QUORUM reads

### PR DESCRIPTION
**Pull request by carlyeks**
For CASSANDRA-9602

### CODE REVIEW

**Comment by mambocab** [comment]
Any reason this shouldn't be added to the other multi-dc tests, in particular `test_network_topology_strategy_users` and `test_network_topology_strategy_counters`?

**Comment by carlyeks** [comment]
Actually, these tests need to be separated because they will only run on 3.0+. I'll post another patch that separates out the each_quorum reads tests.

File: consistency_test.py [hunk 1]
```diff
@@ -482,7 +520,7 @@ def run():
 
     def test_simple_strategy_users(self):
         """
-        Test for a single datacenter, users table.
+        Test for a single datacenter, users table, only the each quorum reads.
```
Comment (mambocab) [inline consistency_test.py]: Is this supposed to be "except for the each quorum reads"?

**Comment by mambocab** [comment]
Other than my note on the comment, everything looks good; ping me and I can merge. Thanks!

**Comment by mambocab** [comment]
Just to follow up: am I right that that comment is incorrect? Happy to fix it myself.

File: consistency_test.py (hunk 1 above)
Comment (iamaleksey) [inline consistency_test.py]: Looks correct to me.

Comment (mambocab) [inline consistency_test.py]: Can you or @carlyeks explain? I guess I'm obtuse today, I don't see it. Looks to me like this test tests everything but each_quorum reads, plus makes sure each quorum calls don't fall over, then [test_simple_strategy_each_quorum_users](https://github.com/riptano/cassandra-dtest/pull/606/files#diff-8e908af48367adae7218567e819d6ec1R553) tests only the each_quorum reads.

**Pull request merged**